aws_lambda_powertools/utilities/parameters/appconf.py

```diff
@@ -0,0 +1,185 @@
+"""
+AWS App Config configuration retrieval and caching utility
+"""
+
+
+import os
+from typing import Dict, Optional, Union
+from uuid import uuid4
+
+import boto3
+from botocore.config import Config
+
+from .base import DEFAULT_PROVIDERS, BaseProvider
+
+CLIENT_ID = str(uuid4())
```

Comment:
__Question__: should we keep this configurable when initializing the class?

Reply:
is that auditable in some way ? if not, i suppose it's best to keep things simple.

Reply:
I think this is a good default, but I'm wondering if there could be cases where someone might want to share a client ID across multiple execution environment.